<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="TrackingWorkflowTerminatedEventArgs.xml" source-language="en-US" target-language="ja-JP">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac5febc14db8feb6659a5e362c3d3a02faed5f57ce7.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">febc14db8feb6659a5e362c3d3a02faed5f57ce7</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs">
          <source>Contains data associated with the termination of a workflow instance.</source>
          <target state="translated">ワークフロー インスタンスの終了に関連したデータが格納されます。</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs">
          <source>A <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs&gt;</ph> is generated by the runtime tracking infrastructure when a workflow instance is terminated.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs&gt;</ph> は、ワークフロー インスタンスが終了するときにランタイム追跡インフラストラクチャによって生成されます。</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs">
          <source>If the <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingProfile&gt;</ph> associated with a workflow instance includes a <ph id="ph2">&lt;xref:System.Workflow.Runtime.Tracking.WorkflowTrackPoint&gt;</ph> configured for a <ph id="ph3">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowEvent.Terminated&gt;</ph><ph id="ph4">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowEvent&gt;</ph>, the workflow tracking infrastructure puts the <ph id="ph5">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs&gt;</ph> in <ph id="ph6">&lt;xref:System.Workflow.Runtime.Tracking.WorkflowTrackingRecord.EventArgs%2A&gt;</ph> in the <ph id="ph7">&lt;xref:System.Workflow.Runtime.Tracking.WorkflowTrackingRecord&gt;</ph> that it sends to the tracking service.</source>
          <target state="translated">ワークフロー インスタンスに関連した <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingProfile&gt;</ph> に、<ph id="ph2">&lt;xref:System.Workflow.Runtime.Tracking.WorkflowTrackPoint&gt;</ph><ph id="ph3">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowEvent.Terminated&gt;</ph> 用に構成された <ph id="ph4">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowEvent&gt;</ph> が含まれる場合には、ワークフロー追跡インフラストラクチャは <ph id="ph5">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs&gt;</ph> を <ph id="ph6">&lt;xref:System.Workflow.Runtime.Tracking.WorkflowTrackingRecord.EventArgs%2A&gt;</ph> 内の <ph id="ph7">&lt;xref:System.Workflow.Runtime.Tracking.WorkflowTrackingRecord&gt;</ph> に格納して追跡サービスに送ります。</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs">
          <source>A workflow instance may be terminated in one of three ways: the host may call <ph id="ph1">&lt;xref:System.Workflow.Runtime.WorkflowInstance.Terminate%2A?displayProperty=nameWithType&gt;</ph>; a <ph id="ph2">&lt;xref:System.Workflow.ComponentModel.TerminateActivity&gt;</ph> activity may be invoked from inside the workflow instance; or an unhandled exception may occur.</source>
          <target state="translated">ワークフロー インスタンスは、ホストが <ph id="ph1">&lt;xref:System.Workflow.Runtime.WorkflowInstance.Terminate%2A?displayProperty=nameWithType&gt;</ph> を呼び出した場合、ワークフロー インスタンスの内部から <ph id="ph2">&lt;xref:System.Workflow.ComponentModel.TerminateActivity&gt;</ph> アクティビティが呼び出された場合、または未処理の例外が発生した場合に終了します。</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs">
          <source>If the workflow is terminated by the host or a <ph id="ph1">&lt;xref:System.Workflow.ComponentModel.TerminateActivity&gt;</ph> activity, the runtime tracking infrastructure sets <ph id="ph2">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs.Exception%2A&gt;</ph> to a <ph id="ph3">&lt;xref:System.Workflow.ComponentModel.WorkflowTerminatedException&gt;</ph> that has its <ph id="ph4">&lt;xref:System.Exception.Message%2A&gt;</ph> property set to a description of the reason for the termination.</source>
          <target state="translated">ホストまたは <ph id="ph1">&lt;xref:System.Workflow.ComponentModel.TerminateActivity&gt;</ph> アクティビティによってワークフローが終了した場合、ランタイム追跡インフラストラクチャは <ph id="ph2">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs.Exception%2A&gt;</ph> を <ph id="ph3">&lt;xref:System.Workflow.ComponentModel.WorkflowTerminatedException&gt;</ph> に設定し、その <ph id="ph4">&lt;xref:System.Exception.Message%2A&gt;</ph> プロパティには終了の理由の説明が設定されます。</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs">
          <source>If the workflow is terminated because of an unhandled exception, the runtime tracking infrastructure passes the unhandled exception in <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs.Exception%2A&gt;</ph>.</source>
          <target state="translated">ワークフローが未処理の例外によって終了した場合は、ランタイム追跡インフラストラクチャは未処理の例外を <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs.Exception%2A&gt;</ph> で渡します。</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs">
          <source><ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs&gt;</ph> is used only by the runtime tracking service to pass information in a <ph id="ph2">&lt;xref:System.Workflow.Runtime.Tracking.WorkflowTrackingRecord&gt;</ph>.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs&gt;</ph> は、ランタイム追跡サービスが <ph id="ph2">&lt;xref:System.Workflow.Runtime.Tracking.WorkflowTrackingRecord&gt;</ph> で情報を渡す場合にのみ使用されます。</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs">
          <source>The data for a <ph id="ph1">&lt;xref:System.Workflow.Runtime.WorkflowRuntime.WorkflowTerminated?displayProperty=nameWithType&gt;</ph> event is passed in a <ph id="ph2">&lt;xref:System.Workflow.Runtime.WorkflowTerminatedEventArgs&gt;</ph>.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Workflow.Runtime.WorkflowRuntime.WorkflowTerminated?displayProperty=nameWithType&gt;</ph> イベントのデータは <ph id="ph2">&lt;xref:System.Workflow.Runtime.WorkflowTerminatedEventArgs&gt;</ph> で渡されます。</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs">
          <source>The following code example demonstrates a method, named <ph id="ph1">`WriteTerminatedEventArgs`</ph>, which captures a <ph id="ph2">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs&gt;</ph>.</source>
          <target state="translated">次のコード例は、<ph id="ph1">`WriteTerminatedEventArgs`</ph> を取得する <ph id="ph2">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs&gt;</ph> という名前のメソッドを示しています。</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs">
          <source>The code checks whether the <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs.Exception%2A&gt;</ph> property is <ph id="ph2">`null`</ph> (<ph id="ph3">`Nothing`</ph> in Visual Basic).</source>
          <target state="translated">このコードは、<ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs.Exception%2A&gt;</ph> プロパティが <ph id="ph2">`null`</ph> (Visual Basic の場合は <ph id="ph3">`Nothing`</ph>) かどうかを確認します。</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs">
          <source>If it is not, the code writes the message associated with the <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs.Exception%2A&gt;</ph> property to the console.</source>
          <target state="translated">そうでない場合は、<ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs.Exception%2A&gt;</ph> プロパティに関連付けられているメッセージをコンソールに書き込みます。</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs">
          <source>If <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs.Exception%2A&gt;</ph> is <ph id="ph2">`null`</ph> (<ph id="ph3">`Nothing`</ph>), the code does not write any exception information to the console.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs.Exception%2A&gt;</ph> が <ph id="ph2">`null`</ph> (<ph id="ph3">`Nothing`</ph>) の場合は、例外情報はコンソールに書き込まれません。</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs">
          <source>This code example is part of the EventArgs Tracking SDK Sample from the Program.cs file.</source>
          <target state="translated">このコード例は、Program.cs ファイルに含まれている EventArgs 追跡の SDK サンプルの一部です。</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs">
          <source>For more information, see <bpt id="p1">[</bpt>EventArgs Tracking Sample<ept id="p1">](http://msdn.microsoft.com/library/94818da2-c3c4-4a13-b47a-a0fb30dbbf1c)</ept>.</source>
          <target state="translated">詳細については、次を参照してください。 <bpt id="p1">[</bpt>EventArgs 追跡のサンプル<ept id="p1">](http://msdn.microsoft.com/library/94818da2-c3c4-4a13-b47a-a0fb30dbbf1c)</ept>です。</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs.Exception">
          <source>Gets the exception that caused the workflow instance to be terminated.</source>
          <target state="translated">ワークフロー インスタンスの終了の原因となった例外を取得します。</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs.Exception">
          <source>The <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph> that caused the workflow instance to be terminated.</source>
          <target state="translated">ワークフロー インスタンスの終了の原因となった <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs.Exception">
          <source>When the workflow instance is terminated because of an unhandled exception, <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs.Exception%2A&gt;</ph> contains the unhandled exception.</source>
          <target state="translated">ワークフロー インスタンスが未処理の例外によって終了する場合は、<ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs.Exception%2A&gt;</ph> に未処理の例外が格納されます。</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs.Exception">
          <source>When the workflow instance is terminated by either a host call to <ph id="ph1">&lt;xref:System.Workflow.Runtime.WorkflowInstance.Terminate%2A?displayProperty=nameWithType&gt;</ph> or by a <ph id="ph2">&lt;xref:System.Workflow.ComponentModel.TerminateActivity&gt;</ph> activity,    <ph id="ph3">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs.Exception%2A&gt;</ph> contains a <ph id="ph4">&lt;xref:System.Workflow.ComponentModel.WorkflowTerminatedException&gt;</ph> that has its <ph id="ph5">&lt;xref:System.Exception.Message%2A&gt;</ph> property set to a description of the reason for the termination.</source>
          <target state="translated">ホストが呼び出した <ph id="ph1">&lt;xref:System.Workflow.Runtime.WorkflowInstance.Terminate%2A?displayProperty=nameWithType&gt;</ph> または <ph id="ph2">&lt;xref:System.Workflow.ComponentModel.TerminateActivity&gt;</ph> アクティビティによってワークフロー インスタンスが終了する場合は、<ph id="ph3">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs.Exception%2A&gt;</ph> に <ph id="ph4">&lt;xref:System.Workflow.ComponentModel.WorkflowTerminatedException&gt;</ph> が格納され、その <ph id="ph5">&lt;xref:System.Exception.Message%2A&gt;</ph> プロパティには終了の理由に関する説明が設定されます。</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs.Exception">
          <source>If the host terminates the workflow instance, it supplies this description in the <ph id="ph1">`string`</ph> parameter to <ph id="ph2">&lt;xref:System.Workflow.Runtime.WorkflowInstance.Terminate%2A?displayProperty=nameWithType&gt;</ph>; if the workflow instance is terminated by a <ph id="ph3">&lt;xref:System.Workflow.ComponentModel.TerminateActivity&gt;</ph>, the description is supplied by <ph id="ph4">&lt;xref:System.Workflow.ComponentModel.TerminateActivity.Error%2A?displayProperty=nameWithType&gt;</ph>.</source>
          <target state="translated">ホストによってワークフロー インスタンスが終了された場合、ホストはこの説明を <ph id="ph1">`string`</ph> の <ph id="ph2">&lt;xref:System.Workflow.Runtime.WorkflowInstance.Terminate%2A?displayProperty=nameWithType&gt;</ph> パラメーターに提供します。<ph id="ph3">&lt;xref:System.Workflow.ComponentModel.TerminateActivity&gt;</ph> によってワークフロー インスタンスが終了された場合、説明は <ph id="ph4">&lt;xref:System.Workflow.ComponentModel.TerminateActivity.Error%2A?displayProperty=nameWithType&gt;</ph> によって提供されます。</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs.Exception">
          <source>The following code example demonstrates a method, named <ph id="ph1">`WriteTerminatedEventArgs`</ph>, which captures a <ph id="ph2">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs&gt;</ph>.</source>
          <target state="translated">次のコード例は、<ph id="ph1">`WriteTerminatedEventArgs`</ph> を取得する <ph id="ph2">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs&gt;</ph> という名前のメソッドを示しています。</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs.Exception">
          <source>The code checks whether the <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs.Exception%2A&gt;</ph> property is <ph id="ph2">`null`</ph> (<ph id="ph3">`Nothing`</ph> in Visual Basic).</source>
          <target state="translated">このコードは、<ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs.Exception%2A&gt;</ph> プロパティが <ph id="ph2">`null`</ph> (Visual Basic の場合は <ph id="ph3">`Nothing`</ph>) かどうかを確認します。</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs.Exception">
          <source>If it is not, the code writes the message associated with the <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs.Exception%2A&gt;</ph> property to the console.</source>
          <target state="translated">そうでない場合は、<ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs.Exception%2A&gt;</ph> プロパティに関連付けられているメッセージをコンソールに書き込みます。</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs.Exception">
          <source>If <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs.Exception%2A&gt;</ph> is <ph id="ph2">`null`</ph> (<ph id="ph3">`Nothing`</ph>), the code does not write any exception information to the console.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs.Exception%2A&gt;</ph> が <ph id="ph2">`null`</ph> (<ph id="ph3">`Nothing`</ph>) の場合は、例外情報はコンソールに書き込まれません。</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs.Exception">
          <source>This code example is part of the EventArgs Tracking SDK Sample from the Program.cs file.</source>
          <target state="translated">このコード例は、Program.cs ファイルに含まれている EventArgs 追跡の SDK サンプルの一部です。</target>       </trans-unit>
        <trans-unit id="125" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowTerminatedEventArgs.Exception">
          <source>For more information, see <bpt id="p1">[</bpt>EventArgs Tracking Sample<ept id="p1">](http://msdn.microsoft.com/library/94818da2-c3c4-4a13-b47a-a0fb30dbbf1c)</ept>.</source>
          <target state="translated">詳細については、次を参照してください。 <bpt id="p1">[</bpt>EventArgs 追跡のサンプル<ept id="p1">](http://msdn.microsoft.com/library/94818da2-c3c4-4a13-b47a-a0fb30dbbf1c)</ept>です。</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>